integration-tests.yml

```diff
@@ -0,0 +1,12 @@
+version: '3.6'
```

Comment:
Why do we need this file?

Reply:
This runs the integration tests in a headless manner so it can be processed by the build/test server in the pipeline.... It's somewhat pre-mature.... it's something I figured we would need on the next steps when hooking these into the pipelines.